src/index.ts

```diff
@@ -274,7 +274,7 @@ function getProgramArgs() {
 async function handleProgramArgs() {
   const { args, parsedOpts } = getProgramArgs();
   if (args.operand === 'export-events') {
-    await exportEventsAsTsv(args.options.file, args.options['overwrite-file']);
```

Comment:
Does this mean we'll no longer be able to overwrite a file? Or will it overwrite by default?

Reply:
Postgres overwrites the given file by default